gulpfile.js

```diff
@@ -0,0 +1,64 @@
+const { src, dest, series, watch } = require('gulp');
+const autoprefixer = require('gulp-autoprefixer');
+const htmlmin = require('gulp-htmlmin');
+const gulpif = require('gulp-if');
+const imagemin = require('gulp-imagemin');
+const uglify = require('gulp-uglify');
+const replace = require('gulp-replace');
+const sass = require('gulp-sass');
+
+function clean(cb) {
+  cb();
```

Comment:
Don't forget to add the cleaning functionality, and take out the js build if not needed